CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ Versioning](http://semver.org/spec/v2.0.0.html).
 
 ### Fixed
 - Check results sent via the agent socket now support handlers.
+- `sensuctl user list` can now output yaml and wrapped-json
```

Comment:
Should this go into `CHANGELOGS/unreleased`?

Reply:
Yes. If that's OK with you, I'll leave it for this PR as I don't think there is any harm in just merging in the Changelog directly this time around, but I'll use the new CHANGELOGS folder in the future.

Reply:
@ccressent don't speak too soon 😆 
<img width="826" alt="screen shot 2019-03-04 at 11 21 01 am" src="https://user-images.githubusercontent.com/10634078/53757089-9db9c400-3e6f-11e9-847b-08d240101ce3.png">